framework/libraries/README.md

```diff
@@ -8,38 +8,39 @@ To build/use any of these (using `new-library` as an example):
 
 2. Navigate to the top-level directory:
 
-	`$ cd new-library`
+    `$ cd new-library`
 
 3. Restore packages by typing the following:
 
-	`$ dotnet restore`
+    `$ dotnet restore`
 	
-[!INCLUDE[DotNet Restore Note](~/includes/dotnet-restore-note.md)]
+    **Note:** Starting with .NET Core 2.0 SDK, you don't have to run [`dotnet restore`](https://docs.microsoft.com/dotnet/core/tools/dotnet-restore) because it's run implicitly by all commands that require a restore to occur, such as `dotnet new`, `dotnet build` and `dotnet run`. It's still a valid command in certain scenarios where doing an explicit restore makes sense, such as [continuous integration builds in Azure DevOps Services](https://docs.microsoft.com/azure/devops/build-release/apps/aspnet/build-aspnet-core) or in build systems that need to explicitly control the time at which the restore occurs.
 	
 		
 4. To build and package the library as a NuGet package, type the following:
 
-	`$ cd src/Library`
-	
-	`$ dotnet build`
-	
-	`$ dotnet pack`
-	
-	Check out the `/bin/Debug` directory to see the generated artifacts and `.nupkg`.
+    ```
+    $ cd src/Library
+    $ dotnet build
+    $ dotnet pack
+    ```
+
+    Check out the `/bin/Debug` directory to see the generated artifacts and `.nupkg`.
 
 5. To run unit tests (only applicable to `new-library`):
 
-	`$ cd ../../test/LibraryTests`
-	
-	`$ dotnet test`
+    ```
+    $ cd ../../test/LibraryTests
+    $ dotnet test
+    ```
 
 And that's it!
 
 ## new-library
 
 The project under `/new-library` targets **only** .NET Core. For that reason,
 this project is stored under the core project directory, so our build server builds it on
-all platforms. Look under https://github.com/dotnet/samples/tree/master/core/libraries/new-library/.
+all platforms. Look under https://github.com/dotnet/samples/tree/master/core/libraries/new-library/. <!-- TODO: THIS LINK IS DEAD !! -->
```

Comment:
The link here needs to be updated, I don't know what it should be currently.

Reply:
After browsing in the history, found that it was deleted in #494.
Should they be added back ? @mairaw .
If so, should I add them back in this PR or a separate one ?

Reply:
And also a small note @mairaw, similar cases should be taken into consideration when addressing #14825, the automated solution should look in samples readme files too, not only look in dotnet/docs and dotnet/dotnet-api-docs.

Reply:
Looking through this, I think the right fix now is to remove the "new-library" section from this readme file.